plugins/catalog-backend/scripts/mock-data

```diff
@@ -5,5 +5,5 @@ curl \
 --header 'Content-Type: application/json' \
 --data-raw '{
     "type": "github",
-    "target": "https://github.com/spotify/backstage/blob/master/plugins/catalog-backend/fixtures/two_components.yaml"
+    "target": "https://github.com/spotify/backstage/blob/master/plugins/catalog-backend/fixtures/example-components.yaml"
```

Comment:
Should it be `examples/example-components.yaml` instead? Feeling like fixtures is more test-oriented name

Reply:
sure, i can change that